persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -511,3 +511,10 @@ moveMessages:
 UPDATE OR REPLACE Message
 SET conversation_id = :to
 WHERE conversation_id = :from;
+
+selectConversationMessagesFromSearch:
+SELECT * FROM MessageDetailsView
+WHERE text LIKE ('%' || :searchQuery || '%')
+AND conversationId = :conversationId
+AND contentType = 'TEXT'
+ORDER BY date DESC;
```

Comment:
I see it has no `LIMIT` or any kind of pagination.

I wonder how performant it would be in case a broad search term is used and there are hundreds of results.

I guess a couple hundred should still be fine.